frontend/src/app/offers/offers-box/offers-box.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+import { OffersService } from '../../homepage-offer/offers.service';
+import { Offer } from '../../homepage-offer/offers.model'
+import { Observable } from 'rxjs/Observable';
+
+@Component({
+  selector: 'volontulo-offers-box',
+  templateUrl: './offers-box.component.html',
+  providers: [OffersService]
+})
+export class OffersBoxComponent implements OnInit {
+  public offers$: Observable<Offer[]>;
+  constructor(private offersService: OffersService) {
```

Comment:
I would add one blank line between class attributes and `constructror`

Also prefer (if method body is empty) to format it like that:
```
constructor(private offersService: OffersService) { }
```